android/AndroidManifest.xml

```diff
@@ -875,6 +874,9 @@
     <meta-data android:name="android.webkit.WebView.EnableSafeBrowsing" android:value="false" />
     <!-- Disable Google's anonymous stats collection -->
     <meta-data android:name="android.webkit.WebView.MetricsOptOut" android:value="true" />
+    
+    <!-- Version >= 3.0. Dex Dual Mode support for Samsung devices -->
+    <meta-data android:name="com.samsung.android.multidisplay.keep_process_alive" android:value="true" />
```

Comment:
Are there some docs about this key? It would be great to insert a link into a comment. And maybe to add necessary comments to other changes too.

Reply:
I got it from the documentation [here](https://developer.samsung.com/samsung-dex/modify-optimizing.html). I will now be adding all the necessary comments for the rest of the code too.